common/app/templates/inlineJS/blocking/polyfills/details.scala.js

```diff
@@ -0,0 +1,49 @@
+@()
+
+(function (window, document) {
+    var weShouldPolyfill = !('open' in document.createElement('details'));
+
+    function toggleDetailsState(details) {
+        if (details.hasAttribute('open')) {
+            details.removeAttribute('open');
+        } else {
+            details.setAttribute('open', 'open');
+        }
+    }
+
+    function handleEvent(event) {
+        var targetNode = event.target;
+
+        while (targetNode.nodeName.toLowerCase() !== 'summary' && targetNode !== document) {
+            targetNode = targetNode.parentNode;
+        }
+        if (targetNode.nodeName.toLowerCase() === 'summary') {
+            toggleDetailsState(targetNode.parentNode);
+        }
+    }
+
+    function bindEvents() {
+        document.addEventListener('click', handleEvent);
+        document.addEventListener('keypress', function (event) {
+            if (event.keyCode === 0x20 || event.keyCode === 0x0D) {
```

Comment:
`keyCode` is deprecated, has been for a long time, although browsers did not implement the standard `key` property until recently. Let's be defensive here and use `keyCode` only as a fallback, no?